instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/config/internal/CommonConfig.java

```diff
@@ -56,6 +57,9 @@ public CommonConfig(InstrumentationConfig config) {
                 new ArrayList<>(HttpConstants.KNOWN_METHODS)));
     statementSanitizationEnabled =
         config.getBoolean("otel.instrumentation.common.db-statement-sanitizer.enabled", true);
+    // TODO change with common conf key
+    queryParameterEnabled =
+        config.getBoolean("otel.instrumentation.jdbc.query-parameter.enabled", false);
```

Comment:
I believe this is not the best place for jdbc specific configuration options. Looking at names of existing configuration options `otel.instrumentation.jdbc.capute-query-parameters` might be a better fit.